main.go

```diff
@@ -55,6 +56,8 @@ var (
 	flagDebug    bool
 	flagDebugDir string
 	flagSeed     seedFlag
+	// TODO(pagran): temporarily mimicry as a flag
+	flagControlFlow = os.Getenv("GARBLE_EXPERIMENTAL_CONTROLFLOW") == "1"
```

Comment:
I assume the plan is to have it on all the time once it's stable?

Reply:
Yes. I think we should leave a flag for users to disable obfuscation, for example to investigate bugs caused by control flow

Reply:
can you please rewrite this TODO without the word "mimicry"? at least it doesn't mean anything to me :)